Lib/test/test_pathlib.py

```diff
@@ -1908,7 +1908,7 @@ def test_resolve_common(self):
 
     @os_helper.skip_unless_symlink
     def test_resolve_dot(self):
-        # See https://bitbucket.org/pitrou/pathlib/issue/9/pathresolve-fails-on-complex-symlinks
+        # See http://web.archive.org/web/20200623062557/https://bitbucket.org/pitrou/pathlib/issues/9/
```

Comment:
I'm not sure this comment really adds much; better to just remove it than use an archive link.

Reply:
The link was added in the [same commit](https://github.com/python/cpython/commit/51af82cc398ed5d68fa2aae09c1baf7ff9fe11e3) than the unit test so, at least when introduced, it meant something. I don't have a strong opinion about this link but I think it would be better to keep it.

Reply:
@serhiy-storchaka, @pitrou Do either of you remember enough context to suggest a replacement for this comment?

Reply:
Well, the link shows in which context the test was introduced in the first place. It doesn't hurt to keep it.